lib/post/utils/post.dart

```diff
@@ -137,6 +139,11 @@ PostView optimisticallyLockPost(PostView postView, bool lock) {
   return postView.copyWith(post: postView.post.copyWith(locked: lock));
 }
 
+// Optimistically marks a post as read. This changes the value of the post locally, without sending the network request
+PostView optimisticallyReportPost(PostViewMedia postViewMedia, bool read) {
```

Comment:
We can remove this and use `optimisticallyReadPost` as mentioned in a previous comment!